src/hotspot/share/utilities/debug.cpp

```diff
@@ -71,6 +71,9 @@ static intx g_asserting_thread = 0;
 static void* g_assertion_context = NULL;
 #endif // CAN_SHOW_REGISTERS_ON_ASSERT
 
+// Set to suppress secondary error reporting.
+bool Debugging = false;
```

Comment:
How would this work? You would set this while debugging? Does it also work in optimized coding or would the value getting optimized away? Maybe make volatile?